---

layout: post
title: References
description: "References and bibliography"
robots: none
date:       2015-11-25 12:00:00
author: guillermodeandajauregui
header-img: "assets/owner/blog/header/gen_post_02.jpg"
thumbnail: /assets/owner/blog/thumbs/thumb02.png
tags: [tag-name-one, tag-name-two]
categories:  [
              2014,
              2015,
              2016,
              2017,
              2018,]


---
<h5>

Here we share articles, bibliography and other interesting sources that have been recently discussed by our work.

<p><br>

<p style="text-align: justify;">

 Gene expression, transcriptomics ...
<p><br>

* http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004224 Geometry of the Gene Expression Space
 of Individual Cells. Discusses the use of polytopes to represent a high dimensional gene expression space.
 <p>

* http://www.nature.com/nbt/journal/v33/n3/full/nbt.3080.html A comprehensive transcriptional portrait of human cancer
 cell lines. Advances in the transcriptomic characterization of cancer cell lines.
 <p>

* http://biodatamining.biomedcentral.com/articles/10.1186/s13040-015-0075-z Non-coding yet non-trivial: a review on the
 computational genomics of lincRNAs. Discusses current knowledge about this class of transcripts.
 <p>

*  http://www.genomebiology.com/2015/16/1/117 quantro: a data-driven approach to guide the choice of an appropriate
normalization method. Data normalization is cardinal to obtain relevant information from high-throughput technologies.
<p>

* http://www.rna-seqblog.com/the-impact-of-normalization-methods-on-rna-seq-data-analysis-2/ The Impact of Normalization
Methods on RNA-Seq Data Analysis. A nice discussion related to normalization and RNA-seq data.
<p>

* http://link.springer.com/chapter/10.1007/978-3-319-21687-4_10 What Makes a Transcriptional Master Regulator? A Systems
Biology Approach. TMRs are a hot focus of research today, given their apparent relation to the rise of not only natural
phenotypes, but pathological ones as well.
<p>

* http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004391 miRTex: A Text Mining System for
 miRNA-Gene Relation Extraction. Recovering miRNA-gene relations is not trivial; text mining may help with it.
 <p><p><p>

<br>
Biological Networks
<p><p><br>

* Three primers on the use of graph theory related to biological networks
<p>
..* http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-8-S6-S8 Graphs in molecular biology<p>
..* http://biodatamining.biomedcentral.com/articles/10.1186/1756-0381-4-10#CR58_42 Using graph theory to analyze biological networks<p><p>
..* http://www.sciencedirect.com/science/article/pii/S0959437X1300138X Genotype to phenotype via network analysis<p>


* http://www.nature.com/ncomms/2015/150722/ncomms8866/abs/ncomms8866.html A draft network of ligand–receptor-mediated multicellular signalling in human. Cell-to-cell communication across multiple cell types
<p>

* http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0123773 Visinets: A Web-Based Pathway Modeling and Dynamic Visualization Tool. A novel tool for pathway visualization.
<p>

* http://www.nature.com/nmeth/journal/v9/n11/abs/nmeth.2212.html A travel guide to Cytoscape plugins. Cytoscape is a great tool for network visualization and analysis. A travel companion to its plugins is a must.
<p>

* http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004462 The Impact of Different Sources of Fluctuations on Mutual Information in Biochemical Networks. Discusses how different sources of variability within biochemical networks impact the interpretation of information transmission.
<p>

* http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0735-5 Differential analysis of biological networks. Differences among phenotypes may be observed in biological networks. Strategies of how to figure them out.
<p>

*http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-323 RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome
<p>

** http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0100335 Batch Effect Confounding Leads to Strong Bias in Performance Estimates Obtained by Cross-Validation
<p><p><p><br>

 Breast Cancer:

<p><p><p><br>

* http://biodatamining.biomedcentral.com/articles/10.1186/s13040-015-0078-9 Iteratively refining breast cancer intrinsic subtypes in the METABRIC dataset. Breast cancer molecular subtyping is a successful application of integrative genomics, and is still an intensely pursued research area.
<p>

* http://onlinelibrary.wiley.com/doi/10.1002/wsbm.109/abstract Estrogen receptor-positive breast cancer: a multidisciplinary challenge. Good starting point to understand estrogen receptor-positive breast cancer
<p>

* http://www.ncbi.nlm.nih.gov/pubmed/26227964 Inhibition of the autocrine IL-6-JAK2-STAT3-calprotectin axis as targeted therapy for HR-/HER2+ breast cancers. Use of regulatory network analysis to propose therapeutic targets.
<p>

* http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004498 Predicting Anticancer Drug Responses Using a Dual-Layer Integrated Cell Line-Drug Network Model.
<p><p><p><p><p><p><br>


 Hallmarks of Cancer:

<p><p><p><p><p><p><br>


* http://www.sciencedirect.com/science/article/pii/S0092867400816839 The Hallmarks of Cancer. Seminal work on the nature of cancer. First paper to be read by anyone starting research on cancer
<p>

* http://www.sciencedirect.com/science/article/pii/S0092867411001279 Hallmarks of Cancer: The Next Generation. Further required reading by the same authors
<p>

* http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3882065/ The aging of the 2000 and 2011 Hallmarks of Cancer reviews: A critique. Some counterpoints to the above articles
<p><p><p><p><br>

 Systems biology and drug research:

<p><p><p><p><br>


* http://www.cell.com/abstract/S0092-8674%2813%2901155-0 The Dynamics of Signaling as a Pharmacological Target. Targeting signaling pathways has always been a major drug mechanism
<p>

* http://bioinformatics.oxfordjournals.org/content/26/12/i246.full Drug-target interaction prediction from chemical, genomic and pharmacological data in an
integrated framework. Drug prediction is a major goal in chemoinformatics
<p>

* http://www.nature.com/nbt/journal/v25/n10/full/nbt1338.html Drug-Target Network. Using network theory for pharmacological research
** http://www.sciencemag.org/content/275/5298/343.full An Information-Intensive Approach to the Molecular Pharmacology of Cancer. The origins of  NCI60.
<p>
* http://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-015-1396-5 Identification of ncRNAs as potential therapeutic targets in multiple sclerosis through differential ncRNA – mRNA network analysis. Studying non-coding RNA and its therapeutic potential.
<p><p><p><p>


<p><p><p><p><p><p><p><br>
R programming resources:
<p><p><p><p><p><p><p><br>



* www.leanpub.com/u/rdpeng : Good introductory material for new R users.
* http://blog.revolutionanalytics.com/2012/02/creating-beautiful-maps-with-r.html : Ideas to integrate information into maps.


</p>
